internal-packages/react-graphql-test-app/package.json

```diff
@@ -5,13 +5,14 @@
   "type": "module",
   "scripts": {
     "build": "rm -rf dist && tsc && vite build",
-    "codegen": "athena-codegen --schema-path schema.graphql --base-dir src/graphql --documents '**/*.graphql' '**/*.tsx' --resolver ./resolver.js",
+    "codegen": "athena-codegen",
```

Comment:
Can we have a CLI argument to actually point to the config file rather than having it be implicit here?

Reply:
the way cosmic config works it that it finds the config with the right format, similar to jest or vite.